packages/salesforcedx-vscode-core/test/commands/forceSourceRetrieve.test.ts

```diff
@@ -70,26 +72,264 @@ describe('Manifest or Sourcepath Gatherer', () => {
       expect.fail('The workspace root path was undefined');
     }
   });
+});
+
```

Comment:
It's worth noting that I am creating/deleting the manifest directory and manifest files. When running the core tests there's a system-tests/assets/sfdx-simple project directory already being created under salesforcedx-vscode/packages (which is  why the vscode.workspace.workspaceFolders is even set). I'm creating an manifest directory before the batch of Manifest Gather tests and deleting it after. For the manifest files, themselves, I'm just creating/cleaning the files before/after each scenario. I did this because it was more efficient than mocking all of the items from fs that I'd need to mock in order to test the additional code paths in the ManifestOrSourcePathGatherer.